bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleEngine.java

```diff
@@ -844,36 +844,46 @@ protected void runRule(RuntimeRule rule, RuleEngineCallbackImpl.TriggerData td)
         }
     }
 
+    protected void runNow(String ruleUID, boolean considerConditions){
```

Comment:
code formatter missing? At least there should be a space before the curly bracket and you have a lot of tabs in here...